DatadogSessionReplay/Sources/Recorder/Utilities/UIImage+SessionReplay.swift

```diff
@@ -0,0 +1,43 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+#if os(iOS)
+import Foundation
+import UIKit
+import DatadogInternal
+import CryptoKit
+
+private var srIdentifierKey: UInt8 = 11
+extension UIImage: DatadogExtended {}
+extension DatadogExtension where ExtendedType: UIImage {
+    var srIdentifier: String {
```

Comment:
suggestion/ add tests for `srIdentifier`

Reply:
covered through UIImageResource tests